aah/list.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Jeevanandam M. (https://github.com/jeevatkm)
+// go-aah/tools source code and usage is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"aahframework.org/essentials.v0"
+)
+
+const aahProjectIdentifier = "aah.project"
+
+var (
+	listCmd = &command{
+		Name:      "list",
+		UsageLine: "aah list",
+		Short:     "List all aah projects in GOPATH",
+		Long: `aah's list command allows you to view all projects that are
+  making use of aah in your GOPATH`,
+		Run: listRun,
+	}
+)
+
+func listRun(args []string) {
+
+	gopath, err := ess.GoPath()
```

Comment:
`gopath` is already validated at [aah.go](https://github.com/go-aah/tools/blob/v0-unstable/aah/aah.go#L61), Can you please remove L#31 to L#36?